src/layer/form-layout/index.md

```diff
@@ -0,0 +1,85 @@
+2. Choose the container type, add a label, and if needed, the number of columns.
+
+   A group can be placed inside a tab or another group.
+   ![QGIS form Add group or tab](./qgis-add-group-tab.jpg "QGIS form Add group or tab")
+   
+3. Drag and drop fields that should be in the tab or the group
+
+To see an example of tabs and groups, you can clone <MerginMapsProject id="documentation/form_setup" />. 
+
+In this project, we created two tabs, *Information* and *Data* and a group *Location* placed in the *Information* tab:
+![QGIS form with tabs and groups](./qgis_forms_layout.jpg "QGIS form with tabs and groups")
+
+The form with tabs and groups will appear in QGIS like this:
+![QGIS form conditional visibility](./qgis_forms_layout_attribute.jpg "QGIS form conditional visibility")
+
+And this is how the same form looks like in <MobileAppName />:
+![Mergin Maps Input form with tabs and groups](./input_forms_layout2.jpg "Mergin Maps Input form with tabs and groups")
+
+
+## Show and hide fields depending on a field value
+Conditional visibility can be applied to groups and tabs, meaning they will be displayed or hidden depending on the value of a field.
+
+We can set the visibility of **Location** tab depending on the value of the **survey** field:
+  - If **survey** is set to **False**, the **Location** tab will not appear in the form,
+  - If **survey** is set to **True**, the **Location** tab will show up in the form.
+
+1. Select the **Location** tab in the **Form Layout** column
+2. Check the **Control Visibility by Expression** option
+3. Type the following expression in the box below: `"survey" IS TRUE`
+4. Background colour can be set in the **Style** tab.
+![QGIS form control visibility by expression](./qgis_forms_layout_visibility.jpg "QGIS form control visibility by expression")
+
+## Display instructions in the form using Text and HTML widget
+Sometimes, you may want to include instructions or tips for surveyors. This can be achieved by using the Text or HTML widgets.
+
+These widgets can be found in *Available Widgets* in the *Other Widgets* section when using the [Drag and Drop Designer](#qgis-drag-and-drop-designer).
+![QGIS HTML and Text Widget](./qgis-form-html-text-widget.jpg "QGIS HTML and Text Widget")
+
+To configure the **Text** widget, enter the text you want to display in the form. On the right, you will see the preview.
+![QGIS Text widget configuration](./qgis-configure-text-widget.jpg "QGIS Text widget configuration")
+
+If you prefer your text to be formatted, you may do so in the **HTML** widget. HTML widget supports these [HTML tags](https://doc.qt.io/qt-6/richtext-html-subset.html#supported-tags)).
+![QGIS HTML widget configuration](./qgis-configure-html-widget.jpg "QGIS HTML widget configuration")
+
+And this is how the Text and HTML widgets look like in the form in QGIS (left) and in the mobile app (right).
+![](./qgis-form-text-html.jpg)
+
```

Comment:
either here or somewhere else we need to document how to use expressions/variables in text/html widget too and add examples for the more complicated setups..